File: features/operational/csat_rating/src/main/java/com/tokopedia/csat_rating/presenter/screenState/SecondScreenState.java
package com.tokopedia.csat_rating.presenter.screenState;


import com.tokopedia.csat_rating.R;

public class SecondScreenState extends ScreenState {

    private String mCaption;
    private String mQuestion;

    public SecondScreenState(String mCaption,String question) {
        this.mCaption = mCaption;
        mQuestion = question;
    }

    @Override
    public int getFirstEmoji() {
        return getSecondEmoji();

    }
    @Override
    public int getSecondEmoji() {
        return R.drawable.rating_active_2;
    }

    @Override
    public String getMessage() {
        return mCaption;
    }

    @Override
    public String getQuestion() {
        return mQuestion;
    }
}

File: features/operational/csat_rating/src/main/java/com/tokopedia/csat_rating/presenter/screenState/SecondScreenState.kt
package com.tokopedia.csat_rating.presenter.screenState


import com.tokopedia.csat_rating.R

class SecondScreenState(private val mCaption: String, private val mQuestion: String) : ScreenState() {

    override fun getFirstEmoji(): Int {
        return getSecondEmoji()

    }

    override fun getSecondEmoji(): Int {
        return R.drawable.emoji_active_2
    }

    override fun getMessage(): String {
        return mCaption
    }

    override fun getQuestion(): String {
        return mQuestion
    }
}
